packages/fxa-settings/src/pages/Signin/index.tsx

```diff
@@ -173,10 +173,18 @@ const Signin = ({
         }
       }
       if (error) {
-        setLocalizedBannerError(
-          getLocalizedErrorMessage(ftlMsgResolver, error)
+        const localizedErrorMessage = getLocalizedErrorMessage(
+          ftlMsgResolver,
+          error.errno === AuthUiErrors.INVALID_TOKEN.errno
+            ? AuthUiErrors.SESSION_EXPIRED
```

Comment:
Can we maybe add a comment about why an invalid token error here should show the session expired message?

(Is it because of this comment in another file that was removed: "If 'invalid token' is received from profile server, it means the session token has expired")